remote.c

```diff
@@ -2163,6 +2163,150 @@ do_del_tsig(RES* ssl, xfrd_state_type* xfrd, char* arg) {
 	send_ok(ssl);
 }
 
+/* returns `0` on failure */
+static int
+cookie_secret_file_dump(RES* ssl, nsd_type const* nsd) {
+	char const* secret_file = nsd->options->cookie_secret_file;
+	char secret_hex[NSD_COOKIE_SECRET_SIZE * 2 + 1];
+	FILE* f;
+	assert( secret_file != NULL );
+
+	/* open write only and truncate */
+	if((f = fopen(secret_file, "w")) == NULL ) {
+		(void)ssl_printf(ssl, "unable to open cookie secret file %s: %s",
+		                 secret_file, strerror(errno));
+		return 0;
+	}
+	for(size_t i = 0; i < nsd->cookie_count; i++) {
+		struct cookie_secret const* cs = &nsd->cookie_secrets[i];
+		ssize_t const len = hex_ntop(cs->cookie_secret, NSD_COOKIE_SECRET_SIZE,
+																 secret_hex, sizeof(secret_hex));
```

Comment:
```suggestion
			secret_hex, sizeof(secret_hex));
```
Indent, nothing important.